### ISSUE TITLE
Clarification on evaluation config for reproduction

### ISSUE THREAD

**Issue by Xiaoqiong-Liu** [wimmerth/anyup#14]
Hi, thanks for the great work on AnyUp!

I’m currently trying to reproduce the VOC segmentation evaluation results. To match the paper, I trained the AnyUp Upsampler using:

```
python train.py
```

with the default config. The full training run and optimizer settings are recorded in `output/2025-11-17/16-03-46/train.log`.

For the probe run, I reused the optimizer hyperparameters from JAFAR’s `eval.yaml` (AdamW, learning rate = 5e-4). While preparing the probe training (`evaluation/train_probes.py`), I moved the scheduler step outside the inner batch loop so that the cosine LR scheduler updates once per epoch rather than every batch:

```python
if self.cfg.sanity and batch_idx == 0:
    break

# Add learning rate to epoch summary
self.scheduler.step()
current_lr = self.optimizer.param_groups[0]["lr"]
self.log_print(
    f"[bold cyan]Epoch {epoch+1} Summary:[/bold cyan] "
    f"Loss = {avg_loss:.6f} - LR = {current_lr:.2e}"
)
```

After this edit, the probe performance on VOC is:

* **Accuracy:** 0.9268
* **IoU:** 0.7311

(Logged in `evaluation/unsupervised/.../train_anyup_seg.log`.)

These scores are noticeably lower than the results reported in the paper, so I’m concerned I may be misunderstanding part of the evaluation setup.

Could you clarify:

1. Are there any other hyperparameters (batch size, learning rate schedule, number of epochs, etc.) that differ between the paper and the public configs?
2. Would it be possible to share the exact `eval.yaml` used in the paper so I can check for mismatches?

Thanks a lot for your help!
